documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -35,7 +39,17 @@ This contract is an ETHBTC synthetic, collateralized in Dai.
 
 ## Prerequisites
 
-Before starting this tutorial you need to clone repo, install the dependencies and compile the smart contracts. To do this run the following:
+Before starting this tutorial you need to clone the repo, install the dependencies, and compile the smart contracts.
+
+### Linux users
+
+If you are using Linux, you may need to first install the following packages before `npm install` can complete properly:
```

Comment:
I think we need to make a note that the `libudev-dev`, `linux-headers-generic`, and `libusb-1.0-0` package names differ between different linux distributions.

Reply:
Will add this, thanks.